synchronization/pipeline.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/synchronization/pipeline.h"
+
+namespace flutter {
+
+//
```

Comment:
Pretty much. Yeah. The comment is to make clang-format not collapse that to a single line :)